CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 ## Unreleased
+* Added support to detect automatically the playground-id when running cli commands in xsoar-6.
```

Comment:
what command is this reffering to? Is there a denistosdk command to run something from the xsoar cli? Otherwise lets specify what exactly this affects

Reply:
Do you think it can be clearer in some way than it is now?